json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
```

Comment:
It's a date that becomes part of the document reference on the framework agreement. It doesn't really make any sense outside of the context of being a thing that goes into the placeholder.  It's formatted differently from dates that go in to the text as dates rather than as part of the document reference number.